libsvmdata/datasets.py

```diff
@@ -13,7 +11,7 @@
 from sklearn.datasets import load_svmlight_file
 
 # TODO make it customizable by the user
-LIBSVMDATA_PATH = pjoin(str(Path.home()), 'libsvm_data/')
+LIBSVMDATA_PATH = Path.home() / 'data' / 'libsvm'
```

Comment:
can you just add a data_home parameter as in https://scikit-learn.org/stable/modules/generated/sklearn.datasets.fetch_openml.html and keep the default to libsvm_data?
so you have the freedom to do it your way?

basically scikit-learn, nilearn, mne they all do the same. They add a xxx_data folder in your home. Why make an exception here.

Reply:
Because this is very annoying when you use `ls` :). I know, I have a TOC about this but this is definetly something that drives me crazy.

Basically, these libraries just did not think about this point and all copied the behavior of the first one. This is not a standard and we can make something which is better with a new library.  

If you consider that we should use a standard, we can put it in `$HOME/.local/share/libsvmdata/`, as specified in the XDG base directory specification.

Reply:
Also, I think the `data_home` is not a solution as most of the time, this fetcher will be call in the internals of a library like `benchopt` where you can't set this parameter.

Using a sensible default value is more important but I am ensure that I can win as there is many library using the `$HOME/xxx_data` or `$HOME/xxx-data`. I would say there is no reason to follow this as it is not a standard, just a practice, but I cannot always win.